### PR TITLE
Add Colors Story

### PR DESCRIPTION
**Pull request by bergieboy**
ezgif com-gif-maker (3)

### CODE REVIEW

**Review by giuliacampana — APPROVED** [review]

File: src/colors/stories/colors.stories.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+};
+
+export const LUKColors = () => {
+  const getTextColor = (hexcolor :string) :string => {
+    const r = parseInt(hexcolor.substr(1, 2), 16);
+    const g = parseInt(hexcolor.substr(3, 2), 16);
+    const b = parseInt(hexcolor.substr(5, 2), 16);
+    const yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000;
+    return (yiq >= 128) ? 'black' : 'white';
+  };
```
Comment (UnsungHero97) [inline src/colors/stories/colors.stories.js:36]: This is pretty cool. Did you consider using the utility from `polished` (no idea if it works well)?

https://polished.js.org/docs/#readablecolor

File: src/colors/stories/colors.stories.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  return (
+    <Card>
+      <CardSegment>
+        <h1>Colors</h1>
+      </CardSegment>
+      <ColorGrid>
+        {
+          Object.entries(Colors).map(([label, colors]) => (
+            (isPlainObject(colors) && !isEmpty(colors))
+              && (
```
Comment (UnsungHero97) [inline src/colors/stories/colors.stories.js:54]: I think you have some unnecessary parenthesis:

```
Object.entries(Colors).map(([label, colors]) => (
  isPlainObject(colors) && !isEmpty(colors) && (
    ...
  )
)
```

File: src/colors/stories/colors.stories.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                            <Typography color="inherit" variant="button">{color}</Typography>
+                            /
+                            <Typography variant="button">{hex}</Typography>
+                          </ColorBlock>
+                        )
+                      ))
+                    }
+                  </div>
+                </CardSegment>
+              )))
```
Comment (UnsungHero97) [inline src/colors/stories/colors.stories.js:75]:
```suggestion
              ))
            )
```

**Review by UnsungHero97 — APPROVED** [review]

File: src/colors/stories/colors.stories.js (hunk 1 above)
Comment (bergieboy) [reply src/colors/stories/colors.stories.js]: it works! I'll use it!

**Review by UnsungHero97 — APPROVED** [review]

**Review by solomontang — APPROVED** [review]

**Pull request merged**